src/visualization/GraphRenderer.h

```diff
@@ -56,19 +59,21 @@ class GraphRenderer{
         sf::Vector2i hoverPosition; ///< a index vector for the actual hovered Node
         sf::Vector2i clickPosition; ///< a index vector for the actual clicked Node
         sf::Color colorOfClickedNode; ///< the color of the Node rectangle, which is in the position of the clickPosition
-        sf::View actualView; ///< a View which we use as main-"view"
-        sf::View defaultView; ///< a View to reset our actualView
         uint maxNodesPerRow; ///< an upper bound for the row lengths
```

Comment:
uint is not a system independent type, for instance, on my computer at home, this does not compile..
can you replace all `uint `occurences with `unsigned`? (you can use a text editor with replace functionality)